common/commands/src/validator/mixnet/delegators/undelegate_from_mixnode.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::context::SigningClient;
+use clap::Parser;
+use log::info;
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    #[clap(long)]
+    pub identity_key: String,
+}
+
+pub async fn undelegate_from_mixnode(args: Args, client: SigningClient) {
+    info!("removing stake from mix-node");
+
+    let res = client
+        .remove_mixnode_delegation(&*args.identity_key, None)
+        .await
+        .expect("failed to remove stake from mixnode!");
+
```

Comment:
maybe for that command it would be useful to also show the pending undelegation event to the user?

Reply:
Yes maybe, think I'll add nice things like that in the next round of improvements. Same could be for showing delays on bonding etc